debian/changelog

```diff
@@ -1,3 +1,16 @@
+landscape-client (24.02+git6339-0ubuntu0) focal; urgency=medium
+
+  * Fixes:
+    - fix bash shell expansion vulnerability
```

Comment:
Please include launchpad bug numbers for ones that have bugs. I know this one does.